src/packages/storybook/withStore/withStore.js

```diff
@@ -52,9 +52,14 @@ class StoreInjector {
     }
 }
 
-const storeInjector = new StoreInjector();
+const storeInjectors = [];
```

Comment:
Good improve! I have a small proposal.
`storeInjectors` is lools like a good candidate for a static variable of `StoreInjector` class.
```
StoreInjector.stores = []
```
What do you think?